<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="ViewsStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QWidget" name="DeviceView">
     <widget class="QPushButton" name="GoToAppViewButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Go To App View</string>
      </property>
     </widget>
     <widget class="QLabel" name="DeviceNameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RaDoTech Device View</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AppView">
     <widget class="QPushButton" name="GoToDeviceViewButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Go to Device View</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="AppStackedWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>541</width>
        <height>551</height>
       </rect>
      </property>
      <widget class="QWidget" name="LoginRegisterPage">
       <widget class="QLabel" name="LoginHeader">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>71</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Login Page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="HomePage">
       <widget class="QLabel" name="HomeHeader">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Home Page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="MeasureNowPage">
       <widget class="QLabel" name="MeasureHeader">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Measure Now Page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="HistoricalPage">
       <widget class="QLabel" name="HistoricalHeader">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Historical Data Page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ProfilePage">
       <widget class="QLabel" name="ProfileHeader">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Profile Page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="VisulizationPage">
       <widget class="QLabel" name="VisulizationHeader">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Visulization Page</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="HomePageButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MeasureNowButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Measure Now</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HistoricalPageButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Historical</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ProfilePageButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Profile</string>
      </property>
     </widget>
     <widget class="QPushButton" name="VisulizationPageButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Visulization</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
